doc/EnvironmentVariables.md

```diff
@@ -0,0 +1,113 @@
+| LocalConfigPath                      | Path to local .json file containing Agent config, defaults to **.\config.json**                                          |
+| MaxRestartCount                      | Max number of restarts allowed before a module is considered to have failed                                              |
+| MetricScrapeInterval                 | Interval at which diagnostic metrics are sampled, defaults to **1 hour**                                                 |
+| MetricUploadInterval                 | Interval at which diagnostic metrics are uploaded, defaults to **1 day**                                                 |
+| Mode                                 | specifies the mode for module deployment, **iotedged / docker / kubernetes**                                             |
+| PerformanceMetricsUpdateFrequency    | Interval to sample system performance metrics, defaults to **5 minutes**                                                 |
+| PersistentVolumeClaimDefaultSizeInMb | Size of the PersistedVolumeClaim, must be used with StorageClassName                                                     |
+| RequestTimeoutSecs                   | Timeout for handling ping and GetTaskStatus direct methods, defaults to **600**                                          |
+| RocksDB_MaxOpenFiles                 | Max number of files to be concurrently opened by RocksDB                                                                 |
+| RocksDB_MaxTotalWalSize              | Max sized to be used by RocksDB's write-ahead-log                                                                        |
+| RunAsNonRoot                         | If set, runs at user = 1000 instead of root                                                                              |
+| RuntimeLogLevel                      | Runtime diagnostic logging level, **fatal / error / warning / info / debug / verbose**, defaults to **info**             |
+| SendRuntimeQualityTelemetry          | Whether to enable sending runtime diagnostics metric, defaults to **true**                                               |
+| Storage_LogLevel                     | RocksDB diagnostic log level, **NONE / FATAL / HEADER / ERROR / WARN / INFO / DEBUG**, defaults to **NONE**              |
+| StorageClassName                     | StorageClassName to be used when creating a PersistedVolumeClaim                                                         |
+| StorageFolder                        | Path to place the EdgeHub database directory, defaults to **TempPath** of the current OS                                 |
+| UpstreamProtocol                     | Protocol used to for upstream connections, defaults to **Amqp**, and falls back to AmqpWs                                |
+| UseMountSourceForVolumeName          | ???, defaults to **false**                                                                                               |
```

Comment:
@darobs I couldn't figure out what this does, can you help me write a short description?

Reply:
Something like this: "Tells the k8s conversion to use the volume mount source as persistent volume name." You can have it refer to the [values.yaml](https://github.com/Azure/iotedge/blob/master/kubernetes/charts/edge-kubernetes/values.yaml#L142) entry for it.